app/components/Text.tsx

```diff
@@ -111,17 +128,50 @@ const $presets = {
 
   bold: [$baseStyle, $fontWeightStyles.bold] as StyleProp<TextStyle>,
 
-  heading: [$baseStyle, $sizeStyles.xxxl, $fontWeightStyles.bold] as StyleProp<TextStyle>,
+  heading: [
+    $baseStyle,
+    $sizeStyles.xxxl,
+    $fontWeightStyles.bold,
+  ] as StyleProp<TextStyle>,
 
-  companionHeading: [$baseStyle, $sizeStyles.md, $fontWeightStyles.medium] as StyleProp<TextStyle>,
+  companionHeading: [
+    $baseStyle,
+    $sizeStyles.md,
+    $fontWeightStyles.medium,
+  ] as StyleProp<TextStyle>,
 
-  subheading: [$baseStyle, $sizeStyles.lg, $fontWeightStyles.medium] as StyleProp<TextStyle>,
+  subheading: [
+    $baseStyle,
+    $sizeStyles.lg,
+    $fontWeightStyles.medium,
+  ] as StyleProp<TextStyle>,
 
   formLabel: [$baseStyle, $fontWeightStyles.medium] as StyleProp<TextStyle>,
 
-  formHelper: [$baseStyle, $sizeStyles.sm, $fontWeightStyles.book] as StyleProp<TextStyle>,
-
-  eventTitle: [$baseSecondaryStyle, $sizeStyles.xxs, $secondaryFontWeightStyles.medium],
+  formHelper: [
+    $baseStyle,
+    $sizeStyles.sm,
+    $fontWeightStyles.book,
+  ] as StyleProp<TextStyle>,
+
+  link: [
+    $baseSecondaryStyle,
+    $sizeStyles.xxs,
+    $secondaryFontWeightStyles.medium,
+  ],
+
+  primaryLabel: [
+    $baseSecondaryStyle,
+    $sizeStyles.xxs,
+    $secondaryFontWeightStyles.medium,
+    { color: colors.palette.primary500, textTransform: "uppercase" },
+  ] as StyleProp<TextStyle>,
```

Comment:
Because of prettier messing with line formatting, I just want to call out that these are the only two that are truly added into this PR